src/LiveComponent/assets/dist/live_controller.js

```diff
@@ -2393,7 +2393,8 @@ class LoadingPlugin {
     }
     getLoadingDirectives(element) {
         const loadingDirectives = [];
-        let matchingElements = element.querySelectorAll('[data-loading]');
+        let matchingElements = [...element.querySelectorAll('[data-loading]')];
+        matchingElements = matchingElements.filter((elt) => elt.closest('[data-controller="live"]') === element);
```

Comment:
This won't work when using multiple controllers, you should use `data-controller*="live"`

Reply:
Oh thank you! Well spotted